lib/util/base64.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+/**
+ * Encodes a string in base64 format.
+ * @param value the string to encode
+ */
+export function encodeString(value: string): string {
+  if (typeof Buffer === "undefined") {
+    return btoa(value);
+  } else {
+    return Buffer.from(value).toString("base64");
+  }
+}
+
+/**
+ * Encodes a byte array in base64 format.
+ * @param value the Uint8Aray to encode
+ */
+export function encodeByteArray(value: Uint8Array): string {
+  if (typeof Buffer === "undefined") {
+    return btoa(new TextDecoder().decode(value));
+  } else {
+    const bufferValue = (value instanceof Buffer) ? value : new Buffer(value);
```

Comment:
new versions of node.js [have deprecated](https://nodejs.org/dist/latest-v8.x/docs/api/buffer.html#buffer_new_buffer_array) the `new Buffer()` syntax. They expect us to use `Buffer.from()` instead.